src/MasterPeace/Bundle/BookBundle/DataFixtures/ORM/LoadQuestionData.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace MasterPeace\Bundle\BookBundle\DataFixtures\ORM;
+
+use Doctrine\Common\DataFixtures\AbstractFixture;
+use Doctrine\Common\DataFixtures\OrderedFixtureInterface;
+use Doctrine\Common\Persistence\ObjectManager;
+use MasterPeace\Bundle\BookBundle\Entity\Book;
+use MasterPeace\Bundle\BookBundle\Entity\Question;
+
+class LoadQuestionData extends AbstractFixture implements OrderedFixtureInterface
+{
+    /**
+     * @return int
+     */
+    public static function getQuestionTotalCount()
+    {
+        return self::getQuestionsPerBook() * LoadBookData::getBookCount();
+    }
+
+    /**
+     * @param ObjectManager $manager
+     */
+    public function load(ObjectManager $manager)
+    {
+        $bookCount = LoadBookData::getBookCount();
+        $questionAmmount = self::getQuestionsPerBook();
+
+        for ($i = 0; $i < $bookCount * $questionAmmount; $i++) {
```

Comment:
for cikle nedarom " $bookCount * $questionAmmount" nes dabar pastoviai daugins ant kiek vieno cklo :)